airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/spec.json

```diff
@@ -12,6 +12,14 @@
         "examples": ["2017-01-25T00:00:00Z"],
         "type": "string"
       },
+      "replication_end_date": {
+        "title": "End Date",
+        "description": "UTC date and time in the format 2017-02-25T00:00:00Z. Any data after this date will not be replicated.",
+        "pattern": "(^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$|default)",
+        "examples": ["default", "2017-02-25T00:00:00Z"],
+        "type": "string",
+        "default": "default"
```

Comment:
I don't think you need to set a `default`, I'd suggest you remove the `default` value and handle it as a `None` value in the config parsing logic. Not required config option default to `None` if they are not set by the user.